cli/command/task.go

```diff
@@ -10,6 +17,220 @@ func NewTaskCommand(b *CommandBase) *TaskCommand {
 	return &TaskCommand{b}
 }
 
-func (e *TaskCommand) Command() cli.Command {
-	return cli.Command{}
+func (t *TaskCommand) Command() cli.Command {
+	return cli.Command{
+		Name:  "task",
+		Usage: "manage layer0 tasks",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Usage:     "create a new task",
+				Action:    t.create,
+				ArgsUsage: "ENVIRONMENT NAME DEPLOY",
```

Comment:
[nit] We were having a discussion about the `ArgsUsage` `NAME` becoming `TASK_NAME`. I think this is the currently agreed consensus on the discussions I saw about this.